smoke-tests/smoke-sdk-grpc-ts.bats

```diff
@@ -39,7 +39,8 @@ teardown_file() {
 
 @test "Manual instrumentation produces span with name of span" {
 	result=$(span_names_for ${TRACER_NAME})
-	assert_equal "$result" '"sleep"'
+	assert_equal "$result" '"sleep"
+"request-handler"'
```

Comment:
This looks out of place

Reply:
Our smoke tests run based on the traces from our example app; this example app has a sleep span and a request-handler span, so this smoke test is asserting both are there.

The alternative to this is to change this smoke test to only make an assertion on the first span name instead of looking at all span names... but I was having some trouble getting that done. The other alternative is to remove this example of tracing an exception.